OpenEdXMobile/res/layout/loading_indicator.xml

```diff
@@ -6,7 +6,9 @@
         android:id="@+id/loading_indicator"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
+        android:layout_centerInParent="true"
         android:layout_gravity="center"
         android:visibility="gone"
+        tools:ignore="ObsoleteLayoutParam"
```

Comment:
What's the background for this change? Shouldn't this be in a separate commit of its own?

Reply:
The change was committed in this [PR](https://github.com/edx/edx-app-android/pull/1561) (RoboGuice 1/3) and was  just reverted in this.

Reply:
ok, it'll be better if this revert is in its own separate commit.